### DISCUSSION TITLE
v1.0.0 🔮

### DISCUSSION THREAD

**Discussion by danpacho** [danpacho/tailwindest#9] [Announcements]
# 🔮 First version just came out!

**Enjoy full typed tailwind with `tailwindest`**

Tell me about your development experience with `tailwindest`! 

> 🪴 Comments are always welcome

<hr /><em>This discussion was created from the release <a href='https://github.com/danpacho/tailwindest/releases/tag/1.0.0'>v1.0.0 🔮</a>.</em>

**Comment by chrisarts** [comment]
Thanks for this buddy ! this will be really helpful on my lib !

**Reply by danpacho** [reply]
Thank you for your feedback!
Happy coding with fully typed tailwind!

**Comment by RobertPakko** [comment]
Hello. I used this package to create a personal website. I made around half the website with vanilla tailwind and got frustrated with the lack of type-safety so I switched to this for the second half.

Using this package was a MAJOR improvement. I was able to move so much faster and everything felt much more organized. I made far fewer mistakes. Really, thank you so much for making this. It's a great tool.

**Reply by danpacho** [reply]
Thanks for the feedback, `tailwind` and `typescript` are awesome! If you think of any good features, please raise an issue🙂!
